core/src/main/java/bisq/core/app/BisqHeadlessApp.java

```diff
@@ -43,7 +43,7 @@
     @Setter
     private GracefulShutDownHandler gracefulShutDownHandler;
     private boolean shutDownRequested;
-    private BisqSetup bisqSetup;
+    protected BisqSetup bisqSetup;
```

Comment:
Why do you need this to be protected instead of private?

Reply:
So that it can be used by subclasses.